docs/release-notes/index.md

```diff
@@ -21,6 +21,44 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Fixes [logstash-next-fixes]
 % *
 
+## 9.2.0 [logstash-9.2.0-release-notes]
+
+### Features and enhancements [logstash-9.2.0-features-enhancements]
+
+##### Event compression for persisted queue
+
+* Support for event compression in the persisted queue, controlled by the per-pipeline `queue.compression` setting, which defaults to `none` [#18121](https://github.com/elastic/logstash/pull/18121)
+* Added metrics for pq compression [#18227](https://github.com/elastic/logstash/pull/18227)
+
+##### Batch performance metrics
+
+* Implements current batch event count and byte size metrics [#18160](https://github.com/elastic/logstash/pull/18160)
+* Implements average batch event count and byte size metrics. The collection of such metric could be disabled, enabled for each batch or done on a sample of the total batches [#18000](https://github.com/elastic/logstash/pull/18000)
+
+
+* Dropped the persistent queue setting queue.checkpoint.interval [#17759](https://github.com/elastic/logstash/pull/17759)
+* Reimplements BufferedTokenizer to leverage pure Java classes instead of use JRuby runtime's classes [#17229](https://github.com/elastic/logstash/pull/17229)
+* Support for user defined metrics [#18218](https://github.com/elastic/logstash/pull/18218)
+* Logging improvement while handling exceptions in the pipeline, ensuring that chained exceptions propagate enough information to be actionable. [#17935](https://github.com/elastic/logstash/pull/17935)
+* [Support for using ES|QL queries](https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/194) in the Elasticsearch filter to add improved flexibility when ingesting data from Elasticsearch is now in Technical Preview. 
+* Gauge type metrics, such as current and peak connection counts of Elastic Agent, are now available in the `_node/stats` API response when the `vertices=true` parameter is included. These metrics are particularly useful for monitoring {{ls}} plugin activity on the {{ls}} Integration dashboards [#18090](https://github.com/elastic/logstash/pull/18090)
+* Improve logstash release artifacts file metadata: mtime is preserved when buiilding tar archives [#18091](https://github.com/elastic/logstash/pull/18091)
+
+### Plugins [logstash-plugin-9.2.0-changes]
+
+**Elastic_integration Filter - 9.2.0**
+
+404: Not Found
+
+**Translate Filter - 3.5.0**
+
+* Introduce opt-in "yaml_load_strategy => streaming" to stream parse YAML dictionaries [#106](https://github.com/logstash-plugins/logstash-filter-translate/pull/106)
```

Comment:
```suggestion
* Introduce opt-in "yaml_load_strategy => streaming" to stream parse YAML dictionaries. This can hugely reduce the memory footprint when working with large YAML dictionaries.  [#106](https://github.com/logstash-plugins/logstash-filter-translate/pull/106)
```